code/linalg/linalg.h

```diff
@@ -34,6 +34,7 @@ extern mp_obj_module_t ulab_linalg_module;
 MP_DECLARE_CONST_FUN_OBJ_KW(linalg_size_obj);
 MP_DECLARE_CONST_FUN_OBJ_1(linalg_inv_obj);
 MP_DECLARE_CONST_FUN_OBJ_2(linalg_dot_obj);
+MP_DECLARE_CONST_FUN_OBJ_2(linalg_norm_obj);
```

Comment:
And finally, here.

Reply:
In 'linalg.h' the functions aren't ordered at all. 'norm' is already after 'dot.
What to do ?

Reply:
That's right, but the header file doesn't really matter. The documentation will be generated from the //| comments, which are in the .c file. So it is enough, if you move the function and its documentation in linalg.c.